js/site_nav.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+/* global require, module, document */
+
+function SiteNav(selector) {
+  this.$body = $(selector);
+  this.$dropdowns = this.$body.find('ul ul');
+  this.$toggles = this.$body.find('.js-toggle');
+  this.assignAria();
+  this.$toggles.change(this.onChange);
+}
+
+SiteNav.prototype.assignAria = function() {
+  var $subLists = this.$body.find('ul ul'),
+      $links = this.$body.find('.js-nav-drop-link');
+
+  this.$body.attr('aria-label', 'Site wide navigation');
+
+  $subLists.each(function() {
+    $(this).attr('aria-label', 'submenu');
+    $(this).attr('aria-hidden', true);
+  });
+
+  $links.attr('aria-haspopup', true);
+}
+
+SiteNav.prototype.onChange = function(ev) {
```

Comment:
This is a personal preference, but I like names like `handleChange` more than names like `onChange`. To me, `onChange` sounds like a method you might call to bind a change callback, but this method is the behavior we carry out to handle a change event. Your call, I'm just being obsessive.